ubuntu_20.04-x86_64-coverity-linux-generic/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04
+
+ARG COVERITY_TOKEN
+
+ENV COVERITY_PROJECT=odp
+
+RUN apt-get update
+
+RUN apt-get install -yy --no-install-recommends \
+  software-properties-common \
+  dirmngr \
+  gnupg-agent
+
+RUN apt-get update --fix-missing
+
+RUN apt-get install -yy \
+  autoconf \
+  automake \
+  ccache \
+  clang \
+  curl \
+  gcc \
+  gcc-10 \
+  git \
+  libcli-dev \
+  libconfig-dev \
+  libcunit1-dev \
+  libpcap-dev \
+  libssl-dev \
+  libtool \
+  libstdc++-10-dev \
+  make \
+  sudo
+
+RUN curl -d token="${COVERITY_TOKEN}&project=${COVERITY_PROJECT}" -o /coverity_tool.tgz https://scan.coverity.com/download/linux64 && \
```

Comment:
The first quotation mark is in wrong place. Should be: `curl -d "token=${COVERITY_TOKEN}&project=${COVERITY_PROJECT}"`

Reply:
Ok, will fix.